packages/pwa-buildpack/lib/WebpackTools/configureWebpack/getModuleRules.js

```diff
@@ -118,7 +118,10 @@ getModuleRules.css = async ({ hasFlag, mode, paths }) => ({
                 {
                     loader: 'css-loader',
                     options: {
-                        localIdentName: '[name]-[local]-[hash:base64:3]',
+                        localIdentName:
+                            mode === 'development'
```

Comment:
That's a nice change but I am sure the cypress tests need to be updated because they depend on the classname's regex to identify an element while testing.

Reply:
@revanth0212 maybe we should use data-attribute for cypress bind test on classNames is not recommend by cypress itself 
https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
We had this discussion with the team a while ago during the dev time and the team decided to use class names instead of data attributes because we dint want to pollute the app with data attributes just for cypress given cypress will mostly be used internally. 

We did have the thought of stripping off data attributes in prod mode but that will not let us run cypress against `develop`, 'release.*`, and `master` instances which we use in regression.

So we settled with class names and decided to use data attributes only if finding an element with class names is getting too complex.

Reply:
**Just my 2 cents:**
I feel like classNames are to brittle, we should use test attributes and strip them in production. I understand your point @revanth0212 i've had those same issues but we should only strip them from the final product that is actually being deployed to customers, I assume this would need to changes from dev-ops.

Reply:
Having data-testid attributes in the final build is also not that bad beyond taking up a few chars they don't do much.  May even be nice to be able to run your smoketests automatically against the live site.